core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/theme/Theme.java

```diff
@@ -1465,4 +1465,12 @@
     String gitConsoleStagedFilesColor();
     String gitConsoleUnstagedFilesColor();
     String gitConsoleErrorColor();
+
```

Comment:
What about having styles specific to VCS and not to Git and Subversion ?
So in this case, replace above and below lines with:
``` java
String vcsConsoleStagedFilesColor();
String vcsConsoleUnstagedFilesColor();
String vcsConsoleErrorColor();
String vcsConsoleModifiedFilesColor();
String vcsConsoleChangesLineNumbersColor();
```